pokemon-checker/src/DataTransferObjects/PokemonDTO.ts

```diff
@@ -10,6 +10,7 @@ export type PokemonConstructorOptions = {
   stats: {
     base_stat: number;
   }[];
+  url?: string;
```

Comment:
We're now expecting that some Pokemon DTOs can be created with a URL (when we generate the name stubs on page load).